HighEnergyObsCoreExt.tex

```diff
@@ -82,7 +82,7 @@
 \begin{document}
 
 \begin{abstract}
-This is a proposed extension to the ObsCore specification for data description, discovery and selection of High Energy Astrophysics (HEA) data, and includes proposed updates to the data product vocabulary, UCDs, and MIME-types to support discovery of HEA data.  
+This is a proposed extension to the ObsCore specification for data description, discovery and selection of High Energy Astrophysics (HEA) data, and includes proposed updates to the data product vocabulary, UCDs, and MIME-types to support discovery of HEA data.
```

Comment:
General comment:  you use `\gls{HE} data` in parts of the document and just `HEA data` elsewhere. Maybe just pick one and stick to it (preferably one with a glossary entry)?